### PR TITLE
Add timestamps to logs

### PR DESCRIPTION
**Pull request by anton-ryzhov**
Add timestamps to logs

### CODE REVIEW

**Comment by anton-ryzhov** [comment]
:+1:

File: kube_log_watcher/main.py [hunk 1]
```diff
@@ -47,8 +49,8 @@ def get_container_label_value(config, label) -> str:
         io.kubernetes.pod.name
     """
     labels = config['Config']['Labels']
-    for l, val in labels.items():
-        if l.endswith(label):
```
Comment (christianberg) [inline kube_log_watcher/main.py:51]: How is `i` better than `l`? 😄 I would understand `k`...
I don't really care either way, just curious why you bothered to change this in this PR.

File: tests/test_scalyr.py [hunk 2]
```diff
@@ -915,7 +915,6 @@ def test_parse_scalyr_sampling_rules(monkeypatch, scalyr_env, fx_scalyr):
         'scalyr_sampling_rules': SCALYR_SAMPLING_RULES,
     })
 
-    print(agent.scalyr_sampling_rules)
```
Comment (christianberg) [inline tests/test_scalyr.py:918]: Why print within a test case?

**Comment by christianberg** [comment]
👍

File: kube_log_watcher/main.py (hunk 1 above)
Comment (anton-ryzhov) [reply kube_log_watcher/main.py:51]: Flake (new version? there wasn't this problem before) in general doesn't like short names, but it's fine with `i`. `label` is already in use here, I didn't come up with better name

File: tests/test_scalyr.py (hunk 2 above)
Comment (anton-ryzhov) [reply tests/test_scalyr.py:918]: That's why I'm removing it. Accidentally found this it the code

Comment (christianberg) [reply tests/test_scalyr.py:918]: I obviously can't tell the difference between red and green and need to practice reading diffs more. 🤦

Comment (anton-ryzhov) [reply tests/test_scalyr.py:918]: Test your color vision maybe 😁

image

**Pull request merged**